src/components/FileElement/File.tsx

```diff
@@ -0,0 +1,32 @@
+import { Dropdown } from "../DropdownMenu/Dropdown";
+import { FileText } from "lucide-react";
+
+interface Props {
+  fileName: string;
+  fileExtension: string;
+}
+
+export function File({ fileName, fileExtension }: Props) {
+  const handleClick = () => {
+    console.log("File clicked!");
+  };
+
+  const handleDropdownClick = (event: React.MouseEvent<HTMLDivElement>) => {
+    event.stopPropagation();
+  };
+
+  return (
+    <div
+      className="relative left-2 top-2 box-border flex h-32 w-32 cursor-pointer flex-col rounded-lg border-2 bg-white p-5"
```

Comment:
Avoid using position to add spacing (`left-2`, `top-2`). Consider adding padding to the parent container.